blog/templates/add_post.html

```diff
@@ -1,10 +1,8 @@
 {% extends 'base.html'   %}
 
-
 {% block content %}
 {% if user.is_superuser %}
-    <h1>Add Post</h1>
-    <br/><br/>
+    <div class="text title"><h1>Add Post</h1></div>
```

Comment:
h1 tag can be in new line it will be more readable

Reply:
done